test/functionalTest/cases/1367_time_measures/time_measure_stats.test

```diff
@@ -31,7 +31,7 @@ brokerStart CB 101,38 IPv4 -reqTimeStat
 --SHELL--
 
 #
-# 01. Initial call to GET /statistics (gives NO time stats as LAST /statistics is excluded)
+# 01. Pre-call to GET /statistics to give VALGRIND a possibility
```

Comment:
What does mean "to give VALGRIND a possibility"? It should be clarified in the comment...